client/my-sites/ads/form-earnings.jsx

```diff
@@ -194,31 +218,42 @@ class AdsFormEarnings extends Component {
 
 		return (
 			<div className="ads__module-content-text module-content-text module-content-text-info">
-				<p>{ translate( 'Payments can have the following statuses:' ) }</p>
-				<ul className="ads__earnings-history-statuses-list">
-					<li className="ads__earnings-history-status">
-						<strong>{ translate( 'Unpaid:' ) } </strong>
-						{ translate( 'Payment is on hold until the end of the current month.' ) }
-					</li>
-					<li className="ads__earnings-history-status">
-						<strong>{ translate( 'Paid:' ) } </strong>
-						{ translate( 'Payment has been processed through PayPal.' ) }
+				<p>
+					<strong>{ translate( 'Ads Served' ) } </strong>
```

Comment:
Hi! I've found a possible matching string that has already been translated **49** times:
`translate( 'No ads' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).